src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -355,11 +355,22 @@ private StageWithCaseData getStageWithCaseData(UUID caseUUID, UUID stageUUID) {
         }
     }
 
+    private Stage getBasicStage(UUID caseUUID, UUID stageUUID) {
+        log.debug("Getting Stage: {} for Case: {}", stageUUID, caseUUID);
+        Stage stage = stageRepository.findBasicStageByCaseUuidAndStageUuid(caseUUID, stageUUID);
+        if (stage != null) {
+            log.info("Got Stage: {} for Case: {}", stageUUID, caseUUID);
+            return stage;
+        } else {
+            throw new ApplicationExceptions.EntityNotFoundException(String.format("Stage UUID: %s not found!", stageUUID), STAGE_NOT_FOUND);
+        }
+    }
+
     private Stage getActiveBasicStage(UUID caseUUID, UUID stageUUID) {
-        log.debug("Getting Base Stage: {} for Case: {}", stageUUID, caseUUID);
```

Comment:
Would having this day ‘Getting active stage’ make it clearer when searching in Kibana?